modules/caddytls/connpolicy.go

```diff
@@ -244,14 +244,18 @@ type ClientAuthentication struct {
 	// which are not in this list will be rejected.
 	TrustedLeafCerts []string `json:"trusted_leaf_certs,omitempty"`
 
+	// requires a client certificate, but will not verify it
```

Comment:
Flush out this godoc comment more. Will need much more explanation! What does it do? What are the possible values? What do they do? I can write this if you want a suggestion.